plugins/mongo_db/mongo_db_state.cpp

```diff
@@ -239,7 +242,40 @@ namespace mongo_db {
     }
 
     auto state_writer::operator()(const transfer_operation& op) -> result_type {
-        
+        auto oid = std::string(op.from).append("/").append(op.to)
+            .append("/").append(state_block.timestamp);
```

Comment:
This isn't unique.
There are no limits to transfer more than 1 time per block from one account to another.
Transfers don't have update operations, so _id can be calculated by Mongo.